pilot/pkg/kube/inject/testdata/inject/resource_annotations.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
```

Comment:
just to confirm: the new files under `/testdata/inject` are identical to the ones under `/testdata/webhook`?

Reply:
Yeah, it's same with the one under `/testdata/webhook`, but some of other files are also same. Do we need to fix all of them?